cypress/integration/journalistCanCreateArticles.feature.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef */
+describe("Journalist can create articles", () => {
+  before(() => {
+    cy.server();
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/auth/sign_in",
+      response: "fixture:journalist_can_login.json",
+      headers: {
+        uid: "journalist@mail.com",
+      },
+    });
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/articles",
+      response: {},
```

Comment:
You expect to see a message but you are stubbing out an empty response, from where is the message coming?

Reply:
Added message in response